src/main/java/academic/GradeBook.java

```diff
@@ -10,6 +10,11 @@ public static void addGrade(String[] args, ArrayList<Grade> currentGrades) {
         currentGrades.add(new Grade(args[0],Integer.parseInt(args[1]),args[2]));
     }
 
+    public static void deleteGrade(Integer indexToBeDeleted, ArrayList<Grade> currentGrades) {
+        currentGrades.remove(indexToBeDeleted - 0);
+        //TODO: Fix remove() not recognising index
```

Comment:
You can look at my RemoveLinkCommand class evaluateLinkNumber to add the exceptions!

Reply:
Alright thanks!